core/defs_hues.scp

```diff
@@ -18,17 +18,31 @@ color_snow                   1153
 color_rock                   1154
 color_spectral               16385
 
+// Ores
+color_o_iron       	0 //option: 0763
+color_o_dull_copper 060a
+color_o_shadow     	0770
+color_o_copper     	0641 //option: 060c
+color_o_bronze     	06d6
+color_o_gold       	045e //option: 084d
+color_o_agapite    	0400
+color_o_verite     	07d1
+color_o_valorite   	0515
+
```

Comment:
Could we add the old Sphere mineral in comment? Rusty, blackrock, bloodrock, etc. Some admin will try to reproduce old sphere. Having here the old color could be usefull.